django/test/utils.py

```diff
@@ -640,3 +640,14 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def tag(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
```

Comment:
@mrbox Hmm, personally I would probably decorate classes rather than use a class attribute, just so I can more easily grep for `@tag` in my test suite and find all occurrences of tagging. In any case, I don't see a reason not to rename this var `func_or_class` or just `obj`, to make it clearer that it can work for either.